src/host/screenInfo.cpp

```diff
@@ -1472,7 +1472,7 @@ bool SCREEN_INFORMATION::IsMaximizedY() const
         coordCursorHeightDiff.Y = sCursorHeightInViewportAfter - sCursorHeightInViewportBefore;
         LOG_IF_FAILED(SetViewportOrigin(false, coordCursorHeightDiff, true));
 
-        _textBuffer->SetCurrentAttributes(oldPrimaryAttributes);
+        newTextBuffer->SetCurrentAttributes(oldPrimaryAttributes);

```

Comment:
*HUH*. This seems like an unrelated but important bug fix

Reply:
yea this was mentioned here: https://github.com/microsoft/terminal/issues/12567#issuecomment-1057980580 and in the previous reply